ci/recipe/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set data = load_setup_py_data() %}
+{% set version = data.get('version') or 'placehold' %}
+{% set release = '.'.join(version.split('.')[:2]) %}
+
+package:
+  name: q2-gneiss
+  version: {{ version }}
+
+source:
+  path: ../..
+
+build:
+  script: python setup.py install
+
+requirements:
+  build:
+    - python 3.5*
+    - setuptools
+
+  run:
+    - python 3.5*
+    - setuptools
+    - gneiss >=0.3.0
+    - scikit-bio
+    - numpy
+    - pandas
+    - seaborn
```

Comment:
We will need bokeh - is that already bundled with qiime2, or should that be added?